### DISCUSSION TITLE
Goto Alternate?

### DISCUSSION THREAD

**Discussion by dstrelau** [nvim-mini/mini.nvim#906] [Q&A] [answered]
I recently converted nearly all my nvim to mini. It's amazing! One thing I'm missing is "goto alternate file" functionality like a.vim, projectionist.vim, other.nvim. Did I miss a way to do this or would this be a feature request?

**Comment by echasnovski** [comment, marked as answer]
Glad you like it so far!

No, there is no such functionality and likely won't be at its 100%. The main reason is that this would require keeping track of what files should be considered alternative on per language/filetype basis. Which is highly against 'mini.nvim' principles (except if it brings **considerable** quality of life improvement).

I wonder if just suggesting files that have same basename without extension inside current file's directoru is enough here?

**Reply by dstrelau** [reply]
Thanks for the quick response. Totally makes sense. I don't really need much other than some basic `foo.go` <-> `foo_test.go` mappings. I don't use 95% of the stuff in these plugins, which is sort of why I was wondering about a "mini-ish" version of them. Maybe I'll have to learn just enough nvim lua to write a function with the 5% I need. 😄
